cabby/data/metagraph/utils.py

```diff
@@ -103,11 +108,88 @@ def update_osm_map(osm_map: Map,
   print("In update_osm_map: %d nodes in graph after adding." % (
     osm_map.nx_graph.number_of_nodes()))
 
+
+def add_conceptual_nodes_and_edges(graph: nx.Graph,
+                                   poi_map: Dict[Text, int],
+                                   wd_relations: pd.DataFrame):
+  for _, row in wd_relations.iterrows():
+    place_node_id = poi_map[row["place"]]
+    graph.add_edge(row["instanceLabel"], place_node_id)
+    graph.add_edge(place_node_id, row["instanceLabel"])
+
+def construct_human_readable_name(poi_row: pd.Series) -> Text:
```

Comment:
This function name isn't very helpful as it doesn't state what is being made into a name. Perhaps poi_to_name() or something like that?

Reply:
Fixed by b0e9f5b9bd83a9fe1d5ef304d7e120501e55ce20